CRITICAL - 2024-08-23 08:37:36 --> CodeIgniter\Exceptions\FrameworkException: The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 252.
 1 SYSTEMPATH\CodeIgniter.php(252): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(214): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-08-23 08:38:12 --> CodeIgniter\Exceptions\FrameworkException: The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 252.
 1 SYSTEMPATH\CodeIgniter.php(252): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(214): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-08-23 08:38:13 --> CodeIgniter\Exceptions\FrameworkException: The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 252.
 1 SYSTEMPATH\CodeIgniter.php(252): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(214): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-08-23 08:38:31 --> CodeIgniter\Exceptions\FrameworkException: The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 252.
 1 SYSTEMPATH\CodeIgniter.php(252): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(214): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-08-23 06:43:06 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-08-23 08:01:14 --> mysqli_sql_exception: Table 'petz24_db.user_tbl' doesn't exist in engine in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-08-23 08:01:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:14 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:14 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `user_tbl`
WHERE `flag` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 08:01:25 --> mysqli_sql_exception: Table 'petz24_db.user_tbl' doesn't exist in engine in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-08-23 08:01:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:25 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:25 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `user_tbl`
WHERE `flag` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 08:01:29 --> mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-08-23 08:01:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:29 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:29 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:31 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-08-23 08:01:32 --> mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-08-23 08:01:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:32 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:32 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 08:01:33 --> mysqli_sql_exception: Table 'petz24_db.user_tbl' doesn't exist in engine in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-08-23 08:01:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:33 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:33 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `user_tbl`
WHERE `flag` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:34 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-08-23 08:01:36 --> mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-08-23 08:01:36 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:36 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:36 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 08:01:44 --> mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(65): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->insertPet()
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-08-23 08:01:44 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pet_tbl` (`pet_name`) VALUES (:pet_name:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\Workflow_Pet.php(65): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->insertPet()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:44 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pet_tbl` (`pet_name`) VALUES (:pet_name:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 APPPATH\Controllers\Workflow_Pet.php(65): CodeIgniter\Model->insert([...], false)
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->insertPet()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:44 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pet_tbl` (`pet_name`) VALUES (:pet_name:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 APPPATH\Controllers\Workflow_Pet.php(65): CodeIgniter\Model->insert([...], false)
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->insertPet()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 08:01:45 --> mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(65): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->insertPet()
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-08-23 08:01:45 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pet_tbl` (`pet_name`) VALUES (:pet_name:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\Workflow_Pet.php(65): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->insertPet()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:45 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pet_tbl` (`pet_name`) VALUES (:pet_name:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 APPPATH\Controllers\Workflow_Pet.php(65): CodeIgniter\Model->insert([...], false)
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->insertPet()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:45 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pet_tbl` (`pet_name`) VALUES (:pet_name:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 APPPATH\Controllers\Workflow_Pet.php(65): CodeIgniter\Model->insert([...], false)
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->insertPet()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 08:01:53 --> mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(65): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->insertPet()
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-08-23 08:01:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pet_tbl` (`pet_name`) VALUES (:pet_name:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\Workflow_Pet.php(65): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->insertPet()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pet_tbl` (`pet_name`) VALUES (:pet_name:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 APPPATH\Controllers\Workflow_Pet.php(65): CodeIgniter\Model->insert([...], false)
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->insertPet()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:01:53 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pet_tbl` (`pet_name`) VALUES (\'CAt\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pet_tbl` (`pet_name`) VALUES (:pet_name:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 APPPATH\Controllers\Workflow_Pet.php(65): CodeIgniter\Model->insert([...], false)
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->insertPet()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 08:02:39 --> mysqli_sql_exception: Table 'petz24_db.user_tbl' doesn't exist in engine in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-08-23 08:02:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:02:39 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:02:39 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `user_tbl`
WHERE `flag` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 08:03:09 --> mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-08-23 08:03:09 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:03:09 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:03:09 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:03:09 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-08-23 08:03:12 --> mysqli_sql_exception: Table 'petz24_db.user_tbl' doesn't exist in engine in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-08-23 08:03:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:03:12 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:03:12 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.user_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `user_tbl`
WHERE `flag` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user_tbl`
WHERE `flag` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_tbl`
WHERE `flag` = :flag:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Workflow_User.php(11): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_User->getUserCount()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_User))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 08:03:28 --> mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-08-23 08:03:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:03:28 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:03:28 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.pet_tbl' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `pet_tbl`
WHERE `flag` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 08:03:28 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 08:13:27 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 08:16:03 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 08:24:41 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 08:24:49 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 08:26:11 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:06:32 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:07:00 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:08:41 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:08:49 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:10:32 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:11:04 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:11:41 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:11:53 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:13:11 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:15:23 --> ParseError: syntax error, unexpected token "}", expecting ";" or "{"
in APPPATH\Controllers\Workflow_Product_Type.php on line 19.
 1 SYSTEMPATH\Autoloader\Autoloader.php(290): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\projects\\petzadmin\\app\\Controllers\\Workflow_Product_Type.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Workflow_Product_Type')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Workflow_Product_Type')
 4 SYSTEMPATH\CodeIgniter.php(899): class_exists('\\App\\Controllers\\Workflow_Product_Type', true)
 5 SYSTEMPATH\CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 09:15:40 --> mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-08-23 09:15:40 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 2 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:15:40 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 4 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:15:40 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 5 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 09:15:51 --> mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-08-23 09:15:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 2 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:15:51 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 4 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:15:51 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 5 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:16:05 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:17:14 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:17:28 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:17:55 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:18:41 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:18:43 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:19:17 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be public (as in class App\Controllers\BaseController)
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:20:05 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-08-23 09:22:03 --> mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Type.php(29): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-08-23 09:22:03 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 2 APPPATH\Controllers\Workflow_Product_Type.php(29): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:22:03 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 4 APPPATH\Controllers\Workflow_Product_Type.php(29): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:22:03 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 5 APPPATH\Controllers\Workflow_Product_Type.php(29): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:22:19 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:22:54 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:23:21 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:25:31 --> Error: Cannot call constructor
in APPPATH\Controllers\Workflow_Product_Type.php on line 17.
 1 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\Workflow_Product_Type->__construct()
 2 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:26:26 --> ParseError: syntax error, unexpected token "public", expecting end of file
in APPPATH\Controllers\Workflow_Product_Type.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(290): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\projects\\petzadmin\\app\\Controllers\\Workflow_Product_Type.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Workflow_Product_Type')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Workflow_Product_Type')
 4 SYSTEMPATH\CodeIgniter.php(899): class_exists('\\App\\Controllers\\Workflow_Product_Type', true)
 5 SYSTEMPATH\CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 09:27:21 --> mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-08-23 09:27:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 2 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:27:21 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 4 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:27:21 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 5 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 09:27:26 --> mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-08-23 09:27:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 2 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:27:26 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 4 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:27:26 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 5 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 09:33:58 --> mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-08-23 09:33:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 2 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:33:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 4 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:33:58 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 5 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 09:34:04 --> mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-08-23 09:34:04 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 2 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:34:04 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 4 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:34:04 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 5 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 09:34:13 --> mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-08-23 09:34:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 2 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:34:13 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 4 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:34:13 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 5 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 09:34:15 --> mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-08-23 09:34:15 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 2 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:34:15 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 4 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:34:15 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 5 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 09:34:18 --> mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-08-23 09:34:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 2 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:34:18 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 4 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:34:18 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 5 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 09:34:23 --> mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-08-23 09:34:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 2 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:34:23 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 4 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:34:23 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 5 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-08-23 09:38:10 --> mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-08-23 09:38:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 2 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:38:10 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 4 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:38:10 --> [Caused by] mysqli_sql_exception: Table 'petz24_db.prt_tbl' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_type_tbl`
JOIN `prt_tbl` ON `prt_tbl`.`pet_id` = `product_type_tbl`.`pet_id`', [], false)
 5 APPPATH\Controllers\Workflow_Product_Type.php(28): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Type->getProductType()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Type))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-08-23 09:38:35 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:38:38 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:41:48 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:59:31 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 09:59:37 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 10:00:24 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 10:02:46 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-08-23 10:03:33 --> ErrorException: Access level to App\Controllers\Workflow_Product_Type::$db must be protected (as in class App\Controllers\BaseController) or weaker
in APPPATH\Controllers\Workflow_Product_Type.php on line 8.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
